### ISSUE TITLE
When only one method is available in a grpc service, I'm not able to create an operation.

### ISSUE THREAD

**Issue by buehler** [riok/Kreya#104]
**Describe the bug**
When creating operations (inside a folder or on the root): if only one rpc method is available in a service, it is not possible to create the method. I assume, this happens because the method field does not fire some "changed" event.

**To Reproduce**
1. create a proto with 1 service that contains 1 method
2. try to create an operation inside Kreya for it

**Expected behavior**
I'm able to create the method.

**Screenshots**
![image](https://user-images.githubusercontent.com/1067213/191266979-c3261c17-b262-49ac-bc44-6b9fb4a7dc95.png)

**Environment (if possible, copy the information from the error dialog or the About menu):**
 - OS: Win 11
 - Kreya Version 
```json
{
  "kreyaVersion": "1.8.0",
  "platform": "Win32",
  "userAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36 Edg/105.0.1343.42"
}
```

**Comment by CommonGuy** [comment]
Thanks for your bug report, we'll look into this

**Comment by CommonGuy** [comment]
We just released Kreya 1.9, which fixes this behaviour.
